bundler/spec/bundler/definition_spec.rb

```diff
@@ -159,6 +167,9 @@
         DEPENDENCIES
           only_java
 
+        CHECKSUMS
+          only_java (1.1-java)
```

Comment:
Why no checksum here? Because the lockfile has not yet been installed?

Reply:
correct, and we don't use the compact index for `file:` sources :(